frontend/README.md

```diff
@@ -1,32 +1,33 @@
+<p align="center">
+    <a href="https://airy.co/">
+    <img width="350" src="./assets/airy_primary_rgb.svg" alt="Airy Logo" />
+    </a>
+</p>
+
+# [Airy](https://airy.co/)- A Conversational Data Platform to drive sales, improve marketing and provide better service
```

Comment:
I wouldn't want to go for "the airy marketing pitch" in the readme of a subfolder of an open-source project. It reads really strange.